doc/source/tutorial/interpolate/ND_regular_grid.rst

```diff
@@ -103,7 +103,61 @@ controlled by the ``fill_value`` keyword parameter:
     numerical artifacts. Consider rescaling the data before interpolating.
 
 
-Finally, we note that if you are dealing with data on Cartesian grids with
-integer coordinates, e.g. resampling image data, these routines may not be the
-optimal choice. Consider using `scipy.ndimage.map_coordinates` instead.
+.. _tutorial-interpolate_cartesian-grids:
+
+Uniformly spaced data
+=====================
+
+If you are dealing with data on Cartesian grids with integer coordinates, e.g.
+resampling image data, these routines may not be the optimal choice. Consider
+using `scipy.ndimage.map_coordinates` instead.
+
+For floating-point data on grids with equal spacing, ``map_coordinates`` can
+be easily wrapped into a `RegularGridInterpolator` look-alike. The following is
+a bare-bones example originating from `the Johanness Buchner's
+'regulargrid' package <https://github.com/JohannesBuchner/regulargrid/>`_::
+
+    class CartesianGridInterpolator:
+        def __init__(self, points, values, method='linear'):
```

Comment:
I know that this is meant as a simple example, but it would be nice to show how to feed all possible kwargs to `map_coordinates`.

Reply:
The last commit adds a note to this effect, but I'd rather avoid going down a rabbit hole of mapping all various kwargs (boundary conditions, fill_value etc) here.